### PR TITLE
clarifying English for files e* and f*

### PR DESCRIPTION
**Pull request by mattsmi**
Making the English a tad clearer for editing.md, exporting.md, faqs.md, files.md, and filtered-decks.md.
Most changes are quite minor.
Cheers

### CODE REVIEW

**Review by dae — COMMENTED** [review]
Thanks as always Matt, a couple of notes below.

File: src/editing.md [hunk 1]
```diff
@@ -9,19 +9,19 @@ cards, and Anki creates cards for us. Click 'Add' in the main window,
 and the Add Notes window will appear.
 
 The top left of the window shows us the current note type. If it does
-not say “Basic,” then you may have added some note types when you
-downloaded a shared deck. The text below assumes that “Basic” is
+not say "Basic,” then you may have added some note types when you
```
[the remark this answers is not in the document]
Comment (mattsmi) [reply src/editing.md:12]: I'm sorry, I must have missed the other. The documents seem to have mixed content: some have smart quotes others straight ASCII quotes, and others a mixture. I thought converting them to ASCII quotes (also the LCD) would help make the display better. It should also allow the displaying software to interpret them as it pleases. Sorry about missing one!

File: src/filtered-decks.md [hunk 2]
```diff
@@ -150,33 +151,33 @@ Display cards that have the smallest interval first.
...
+that may take some time to get through and you want to review those cards
+first, that you are most in danger of forgetting.
 
 ## Steps & Returning
 
 Please see the section on [learning](studying.md#learning) as a reminder of how
 steps work.
 
-By default, Anki will use the steps of a card’s home deck. If a new card
+By default, Anki will use the steps of a cardss home deck. If a new card
```
[the remark this answers is not in the document]
Comment (mattsmi) [reply src/filtered-decks.md:180]: Finger trouble. It should have been an ASCII quote, not a second 's'! I am very sorry to have wasted your time on that!

File: src/editing.md [hunk 3]
```diff
@@ -76,42 +76,42 @@ or phrase on each side, as soon as you find yourself wanting to include
...
 
-To customize fields, click the “Fields…​” button when adding or editing
+To customize fields, click the "Fields…​” button when adding or editing
 a note, or while the note type is selected in the Manage Note Types
 window.
 
 You can add, remove, or rename fields by clicking the appropriate
-buttons. To change the order in which the fields appear in this dialog
-and the add notes dialog, you can use the reposition button, which asks
+buttons. To change the order in which the fields appear in this dialogue
```
[the remark this answers is not in the document]
Comment (mattsmi) [reply src/editing.md:111]: OK. I've seen both. From now on, I'll make sure to use the dialog spelling.
Thanks!

File: src/editing.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dae) [reply src/editing.md:12]: Not a problem, but there are actually quite a few more below - might be worth just doing a blanket find&replace.

File: src/filtered-decks.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dae) [reply src/filtered-decks.md:180]: Happens to the best of us :-)

File: src/editing.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mattsmi) [reply src/editing.md:12]: Shall do. Do you want me to correct and resubmit this pull request?

[the remark this answers is not in the document]
Comment (dae) [reply src/editing.md:12]: If you could rebase this one with the changes that'd probably be neatest, and then we can do a find&replace on the rest of the files in a separate change

**Comment by mattsmi** [comment]
Have fixed dialog and those smart quotes in the e* and f* markdown files within this pull request.

**Comment by dae** [comment]
Thanks Matt!

**Comment by uuh00** [comment]
Hi, I want to use this https://cssgradient.io/ feature  but it’s not working with anki are this not supported?

**Comment by mattsmi** [comment]
> Hi, I want to use this https://cssgradient.io/ feature but it’s not working with anki are this not supported?

Have you tried using linear-gradient in your CSS? The following is an example, where I colour the text from a language with the colours of its flag, gradually changing from one colour to the other.
_.la-colour {
	 background: rgb(178,181,238);
	background: linear-gradient(90deg, #ffe6e6 0%, #fffbe6 100%); 
}_

**Comment by uuh00** [comment]
I try to use 
mask: radial-gradient(circle, #0000 43%, rgba(0, 0, 0, 0.9) 54%);
    -webkit-mask: radial-gradient(circle, #0000 43%, rgba(0, 0, 0, 0.9) 54%);
    
    I want blurry all image expect the center but its not working with me

**Comment by uuh00** [comment]
> > Hi, I want to use this https://cssgradient.io/ feature but it’s not working with anki are this not supported?
> 
> Have you tried using linear-gradient in your CSS? The following is an example, where I colour the text from a language with the colours of its flag, gradually changing from one colour to the other.
> _.la-colour { background: rgb(178,181,238); background: linear-gradient(90deg, #ffe6e6 0%, #fffbe6 100%); }_

I try to use
mask: radial-gradient(circle, #0 43%, rgba(0, 0, 0, 0.9) 54%);
-webkit-mask: radial-gradient(circle, #0 43%, rgba(0, 0, 0, 0.9) 54%);

I want blurry all image expect the center but its not working with me

**Comment by dae** [comment]
Such questions sound like they belong on https://forums.ankiweb.net/c/card-design/6

**Pull request merged**